starfish/util/exec.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+import unittest
+from typing import Sequence
+
+import jsonpath_rw
+import numpy as np
+import pandas as pd
+
+from starfish.intensity_table import IntensityTable
+from starfish.types import Features
+from starfish.util import clock
+
+
+def stages(commands, subdirs=None, keep_data=False):
```

Comment:
Is this a thing that could be used to refactor out some code from https://github.com/spacetx/starfish/blob/master/starfish/test/full_pipelines/cli/test_iss.py ?

Seems useful to have more generally as we increase the number of CLI tests, which is something that @shanaxel42 is working on right now. Shannon, would you like to review this PR as well?

Reply:
> Is this a thing that could be used to refactor out some code from 

Exactly.

> Seems useful to have more generally as we increase the number of CLI tests

I certainly see adding to gh-516 and gh-519